snuba/utils/streams/consumer.py

```diff
@@ -493,6 +566,7 @@ def build_kafka_consumer_configuration(
 ) -> Mapping[str, Any]:
     return {
         "enable.auto.commit": False,
+        "enable.auto.offset.store": False,
```

Comment:
If you run this entire patch without this one line change on top of getsentry/snuba#644, that test will fail.